src/components/services-view/services-view.controller.ts

```diff
@@ -65,12 +65,17 @@ export class ServicesViewController implements angular.IController {
     }
 
     this.ctrl.currentFilter = category;
-    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || 'all');
+    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || null);
```

Comment:
The not-yet-merged design doc is at https://github.com/cshinn/openshift-origin-design/blob/12bc9f0770fa761a1301b76e9738cfa382236c6e/web-console/1-homepage/catalog.md